source/whats-new.txt

```diff
@@ -26,6 +27,49 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.11:
+
+What's New in 4.11
+------------------
+
+New features of the 4.11 {+driver-short+} release include:
+
+- Recursive Schema Support
+
+  - The driver now supports recursive schemas and provides type safety
+    on dot notation queries up to a depth of 9. Beyond a depth of 9,
+    your code still compiles but is no longer type checked and falls
+    back to a type of ``any``. This depth limit is a product of
+    Typescript's recursive type limitations.
+
+    - Create an interface with a recursive schema:
+
+      .. code-block:: js
+
+         interface CircularSchema {
+             name: string;
+             nestedSchema: CircularSchema;
+         }
+
+    - Suppose we have a collection of type ``Collection<CircularSchema>``.
+      Below a depth of 9, type checking is enforced. The following code
+      results in a compilation error because the name must be a
+      ``string`` type:
+
+      .. code-block:: js
+
+         collection.findOne({ 'nestedSchema.nestedSchema.nestedSchema.name': 25 })
+
+    - At a depth greater than 9, the following code compiles but is not
+      type checked, as it is at a depth of 11:
+
+      .. code-block:: js
+
+         collection.findOne({
+             'nestedSchema.nestedSchema.nestedSchema.nestedSchema.nestedSchema.nestedSchema.nestedSchema.nestedSchema.nestedSchema.nestedSchema.name': 25
```

Comment:
Issue: I think it's difficult to use this example to demonstrate depth of 11.

Suggestion: think of a shorter name for the CircularSchema type if you want to show an example and have the reader be able to count 10 instances of it.